### PR TITLE
integrating pylint test for database

### PR DESCRIPTION
**Pull request by jamzheng109**

### CODE REVIEW

**Review by jshilts657 — APPROVED** [review]
Looks good

**Review by rzheng2019 — APPROVED** [review]
Looks good

File: .github/workflows/current-workflow.yml [hunk 1]
```diff
@@ -452,6 +452,21 @@ jobs:
     - name: Run pylint/pass or fail
       run: pylint ./ --fail-under=5
 
+  Run-Pylint-Database:
+    name: Pylint test database
+    runs-on: ubuntu-latest
+    needs: [Get-Branch-Name, Run-Database-Unit-Tests]
+
+    container:
+      image: ghcr.io/bumetcs673/seprojects-cs673olf24team6/flask_backend:${{ needs.get-branch-name.outputs.sanitized_branch_name }}
```
Comment (rzheng2019) [inline .github/workflows/current-workflow.yml]: This image path is for the flask_backend when it should be the sqlite_database

**Review by rzheng2019 — CHANGES_REQUESTED** [review]

**Review by jshilts657 — COMMENTED** [review]
You need to merge dev and push up as soon as you are able

**Pull request closed without merging**